### ISSUE TITLE
Flag `-d` is missing in "flutter test" for `integration_test`

### ISSUE THREAD

**Issue by nilsreichardt** [flutter/flutter#85453]
## Description
When I try to run an integration test with `flutter test`, is there no option to pass the `-d`, like you could with `flutter driver`.

I have running multiple devices. With `-d` my goal is to select the device, where I want to start the integration test.

But when I run `flutter test integration_test -d emulator-5554` I got this exception:
```
Integration tests and unit tests cannot be run in a single invocation. Use separate invocations of `flutter
test` to run integration tests and unit tests.
```

But when I run `flutter test -d emulator-5554`, is it working (but then are unit and widget tests are executed) 🤔 

#### Flutter Version
```
Flutter 2.3.0-24.1.pre • channel beta • https://github.com/flutter/flutter.git
Framework • revision 615957513e (3 days ago) • 2021-06-25 09:24:44 -0700
Engine • revision 9d517f475b
Tools • Dart 2.14.0 (build 2.14.0-188.5.beta)
```

**Comment by asashour** [comment]
How about?

    flutter -d emulator-5554 test integration_test

which works with me.

However, I see the point of the error when `flutter test integration_test -d emulator-5554` is used, which says

    Integration tests and unit tests cannot be run in a single invocation. Use separate invocations of
    `flutter test` to run integration tests and unit tests.

**Comment by nilsreichardt** [comment]
I can't reproduce it anymore. Seems to be fixed 🙂 I used Flutter 3.0.0
